stumpy/mstump.py

```diff
@@ -459,6 +492,8 @@ def _multi_distance_profile(
         μ_Q[:, query_idx],
         σ_Q[:, query_idx],
         T_subseq_isconstant,
+        Q_subseq_isconstant[:, query_idx].reshape(-1, 1),
```

Comment:
@seanlaw 
Is line above (i.e. line 495) readable? (Note that the output of `np.atleast_2d` is different. That is why I did not use it here)

Reply:
Cool! That is more readable for sure. I will revise the code.